libs/recnet-api-model/src/lib/api/announcement.ts

```diff
@@ -2,12 +2,16 @@ import { z } from "zod";
 
 import { announcementSchema } from "../model";
 
+const booleanSchema = z
+  .union([z.boolean(), z.literal("true"), z.literal("false")])
+  .transform((value) => value === true || value === "true");
+
```

Comment:
nit: Maybe rename it to something like `queryParamsBooleanSchema` since this will only be used when there's boolean params in https request query params. (If it's POST, PATCH, etc., we can use request body and we can use `z.boolean()` redirectly.)

Reply:
fixed